### DISCUSSION TITLE
Star Exchange

### DISCUSSION THREAD

**Discussion by Mattral** [Schweinepriester/achievement-testarea#45] [Show and tell]
Let's exchange stars here

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Reply by KamBRJC** [reply]
star back https://github.com/z-nnovation/GitSwitcher

**Reply by Istituto-freudinttheprodev** [reply]
> Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

@ternera
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Reply by SDBoii** [reply]
@peterbaccaro Could you please check this project out and star it please if you liked it
https://github.com/Sumdiboii/PokeDex-Project

**Comment by Denisyasyuchenya** [comment]
give you a star

star my https://github.com/Denisyasyuchenya/my-location-app

**Reply by Istituto-freudinttheprodev** [reply]
@Denisyasyuchenya
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by Denisyasyuchenya** [reply]
Done

**Reply by Istituto-freudinttheprodev** [reply]
You're the best! Thank you so much! Do you know anyone else I could star exchange with?

**Reply by SDBoii** [reply]
@Denisyasyuchenya Could you please check this project out and star it please if you liked it
https://github.com/Sumdiboii/PokeDex-Project

**Comment by KamBRJC** [comment]
hi give back star https://github.com/z-nnovation/GitSwitcher

**Reply by Istituto-freudinttheprodev** [reply]
@KamBRJC
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Comment by felizardo27** [comment]


**Reply by Istituto-freudinttheprodev** [reply]
@felizardo27
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Reply by Denisyasyuchenya** [reply]
Done star back https://github.com/Denisyasyuchenya/my-location-app

**Reply by Istituto-freudinttheprodev** [reply]
I starred this repo of yours too!

**Reply by SDBoii** [reply]
@felizardo27 Could you please check this project out and star it please if you liked it
https://github.com/Sumdiboii/PokeDex-Project

**Comment by Istituto-freudinttheprodev** [comment]
> Help me get starstruck badge, I will ⭐️star⭐️ your repo back - provide link in the comment please! https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

@Mattral
star mine i stared yours:
https://github.com/Istituto-freudinttheprodev/Sito-web-lavoro-di-gruppo-INFORMATICA

**Comment by giangeralcus** [comment]
Starred several repos above ⭐ (Mattral, ternera, Denisyasyuchenya, peterbaccaro, elijahpereira, ArielMAJ, ganochenkodg, AdityaNow). Star for star please: https://github.com/giangeralcus/gg-tetris - I'll star back anyone who stars me!
